python/lsst/ap/association/transformDiaSourceCatalog.py

```diff
@@ -378,12 +378,13 @@ def unpack(self, input_flag_values, flag_name):
         output_flags : `numpy.ndarray`
             Numpy named tuple of booleans.
         """
-        bit_names_types = self.output_flag_columns[flag_name]
-        output_flags = np.zeros(len(input_flag_values), dtype=bit_names_types)
+        output_flags = np.zeros(len(input_flag_values),
+                                dtype=[(name, bool) for name in self.output_flag_columns[flag_name]])
 
-        for bit_idx, (bit_name, dtypes) in enumerate(bit_names_types):
-            masked_bits = np.bitwise_and(input_flag_values, 2**bit_idx)
-            output_flags[bit_name] = masked_bits
+        for name in self.output_flag_columns[flag_name]:
```

Comment:
You're basically iterating over column names, right? This might be clearer if the iterable were, accordingly, `columnName`, here and in the list comp above, as it is in the `flagExists` method below

Of course, camelCase vs snake_case is rearing its head, maybe it should be `column_name` throughout...

Reply:
They're the name of the thing in the dictionary, so it's just `name`. It's generally better to keep iterator variables short.

And yeah, Chris was unconsistent with camel vs. snakes. 🤷